src/ensembl/production/metadata/api/models/assembly.py

```diff
@@ -57,10 +57,15 @@ class AssemblySequence(Base):
     length = Column(Integer, nullable=False)
     sequence_location = Column(String(10))
     md5 = Column(String(32))
-    sha512t4u = Column(String(128))
-    # One to many relationships
-    # none
-    # many to one relationships
-    # assembly_id within assembly
+    # column need renaming as well
+    sha512t24u = Column(String(128))
     assembly = relationship('Assembly', back_populates="assembly_sequences")
 
+    # backward compatibility with old column name sha512t2u
+    @property
+    def sha512t2u(self):
```

Comment:
Not `sha512t4u`?

Reply:
Well spotted Andres: https://github.com/Ensembl/ensembl-metadata-api/pull/49 But already fixed :-)

Reply:
https://github.com/Ensembl/ensembl-metadata-api/pull/49/files More precisely